### ISSUE TITLE
update horizontal/vertical layout while dropdown is opened causes wrong sizing

### ISSUE THREAD

**Issue by valdrinkoshi** [PolymerElements/iron-dropdown#74]
See http://jsbin.com/woreyo/2/edit?html,output
- click `open`
- click `left` --> correctly updated
- click `top` --> correctly updated
- click `right` --> does not update :x

**Comment by valdrinkoshi** [comment]
This is because the other 2 dimensions don't get reset on `resetFit`.
The fix should be to add this in `resetFit`
```javascript
// Reset the other 2 dimensions
this.style[hAlign === 'left' ? 'right' : 'left'] = 'auto';
this.style[vAlign === 'top' ? 'bottom' : 'top'] = 'auto';

**Comment by valdrinkoshi** [comment]
Was fixed in `iron-fit-behavior v1.1.1`
